attheme-default-values.js

```diff
@@ -0,0 +1 @@
b:{red:219,green:88,blue:92,alpha:255},chat_attachHideIcon:{red:255,green:255,blue:255,alpha:255},chat_attachFileBackground:{red:52,green:160,blue:244,alpha:255},chat_inMediaIcon:{red:255,green:255,blue:255,alpha:255},chat_attachCameraIcon4:{red:180,green:85,blue:224,alpha:255},chats_mentionIcon:{red:255,green:255,blue:255,alpha:255},chat_attachCameraIcon3:{red:67,green:169,blue:242,alpha:255},switch2ThumbChecked:{red:68,green:168,blue:234,alpha:255},chat_attachCameraIcon1:{red:255,green:125,blue:48,alpha:255},chat_emojiPanelBadgeText:{red:255,green:255,blue:255,alpha:255},chat_attachCameraIcon5:{red:97,green:208,blue:97,alpha:255},chat_attachContactBackground:{red:62,green:191,blue:250,alpha:255},chat_attachCameraIcon6:{red:254,green:193,blue:37,alpha:255},switch2Check:{red:255,green:255,blue:255,alpha:255},key_changephoneinfo_changeText:{red:77,green:131,blue:179,alpha:255},chat_attachVideoBackground:{red:227,green:113,blue:121,alpha:255},chat_outMediaIcon:{red:239,green:255,blue:222,alpha:255},chat_attachSendBackground:{red:62,green:191,blue:250,alpha:255},chat_attachContactIcon:{red:255,green:255,blue:255,alpha:255},chat_attachHideBackground:{red:174,green:170,blue:184,alpha:255},chat_attachAudioBackground:{red:246,green:135,blue:81,alpha:255},chat_attachSendIcon:{red:255,green:255,blue:255,alpha:255},chat_emojiPanelBadgeBackground:{red:77,green:166,blue:234,alpha:255},key_graySectionText:{red:138,green:138,blue:138,alpha:255},chat_attachVideoIcon:{red:255,green:255,blue:255,alpha:255},chat_attachGalleryIcon:{red:255,green:255,blue:255,alpha:255},location_sendLiveLocationIcon:{red:255,green:255,blue:255,alpha:255},chat_attachFileIcon:{red:255,green:255,blue:255,alpha:255},chat_attachLocationBackground:{red:63,green:200,blue:122,alpha:255},chat_attachAudioIcon:{red:255,green:255,blue:255,alpha:255},switch2TrackChecked:{red:160,green:214,blue:250,alpha:255},chat_attachCameraIcon2:{red:235,green:88,blue:80,alpha:255},chat_inMediaIconSelected:{red:226,green:248,blue:255,alpha:255},chat_attachGalleryBackground:{red:164,green:122,blue:217,alpha:255},switch2Track:{red:255,green:176,blue:173,alpha:255},chat_attachLocationIcon:{red:255,green:255,blue:255,alpha:255},chat_outMediaIconSelected:{red:212,green:245,blue:188,alpha:255},chat_inContactPhoneSelectedText:{red:47,green:52,blue:56,alpha:255},dialogCameraIcon:{red:255,green:255,blue:255,alpha:255},chats_actionUnreadPressedBackground:{red:242,green:242,blue:242,alpha:255},chats_actionUnreadBackground:{red:255,green:255,blue:255,alpha:255},chat_inAudioPerfomerSelectedText:{red:47,green:52,blue:56,alpha:255},chats_actionUnreadIcon:{red:115,green:115,blue:115,alpha:255},chat_outAudioPerfomerSelectedText:{red:53,green:66,blue:52,alpha:255},chats_onlineCircle:{red:75,green:203,blue:28,alpha:255},chat_status:{red:213,green:232,blue:247,alpha:255},Chat_outSentCheckRead:{red:93,green:176,blue:80,alpha:255},actionBarDefaultSubmemuItemIcon:{red:103,green:107,blue:112,alpha:255}};exports.default=defaultValues;
```

Comment:
Do not commit compiled files. `git` is for managing source code, not compiled files.